scripts/uninstall.sh

```diff
@@ -21,6 +33,43 @@ TMP_FUNCTION_DIR=$(mktemp -d)
 echo -e "\033[1;34mDynatrace function for Google Cloud Platform monitoring / uninstall script"
 echo -e "\033[0;37m"
 
+if ! command -v yq &> /dev/null
+then
+    err 'yq and jq is required to install Dynatrace function. Please refer to following links for installation instructions:
```

Comment:
Minor: "install" in this context might be confusing. Please, change it to uninstall